lib/Lift.cpp

```diff
@@ -546,53 +546,48 @@ CreateConstFromMemory(const uint64_t addr, llvm::Type *type,
     } break;
 
     case llvm::Type::PointerTyID: {
+      const auto pointer_type = llvm::dyn_cast<llvm::PointerType>(type);
+      const auto size = dl.getTypeSizeInBits(type);
+      auto val = ReadValueFromMemory(addr, size, arch, program);
+      result = llvm::Constant::getIntegerValue(pointer_type, val);
```

Comment:
Woah, does `llvm::Constant::getIntegerValue` work on pointer types?

Reply:
It works on pointer types as well. It converts the APint value to pointers.